client/termdb/TermdbVocab.js

```diff
@@ -1171,6 +1171,10 @@ export class TermdbVocab extends Vocab {
 			}
 		})
 	}
+
+	async rungeneORA(body) {
```

Comment:
i prefer not to create this new method in vocab, since geneORA is not dependent on a ds. in geneora code please directly run `dofetch3('genesetOverrepresentation')`